modules/AWS/eks/locals.tf

```diff
@@ -13,9 +13,10 @@ locals {
 
   cluster_service_ipv4_cidr = "172.20.0.0/16"
 
-  workers_additional_policies = var.osquery_secret_name != "" ? [aws_iam_policy.laas[0].arn,aws_iam_policy.fleet_enrollment_secret[0].arn] : []
-
   osquery_secret_region = var.osquery_secret_region != "" ? var.osquery_secret_region : var.region
 
   account_id = data.aws_caller_identity.current.account_id
+
+  eks_node_policies = ["arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore", "arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy", "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly", "arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy"]
```

Comment:
If necessary, we can make it configurable in tfvars to let pass any custom policies to be attached to nodegroup iam role

Reply:
/nit we can break it into new lines

I think the configurabitlity is a good idea but let's wait when we know it is useful. It could be good to raise this as an improvement for Terraform.

Reply:
Yes, multi line list looks better. Fixed